### ISSUE TITLE
Custom Filters for Columns

### ISSUE THREAD

**Issue by l0psec** [redcanaryco/mac-monitor#15]
**At a high level -- can you summarize your request?**
The ability to add custom filters against Columns that can match against a value. This could be for any column including Process Name, Command Line, Effective user, Source process, etc. 

**What is the current alternative solution?**
There does not appear to be a way to create custom filters for values of interest. Currently, you can filter by finding a value and right clicking to then add a filter. 

**Anything else?**
It would be awesome to have different operators for these values that could be used to match. For example: CONTAINS, BEGINS WITH, MATCHES (regex), DOES NOT CONTAIN.

**Comment by Brandon7CC** [comment]
Hey @l0psec!

This is a wonderful idea and is something I think would be awesome to add, but due to my current backlog and what I'm thinking might be more than a few headaches technically / design wise I don't think this is something to expect in the near future 😞
